conda_build/api.py

```diff
@@ -57,7 +57,7 @@ def check(recipe_path, no_download_source=False, config=None, **kwargs):
 
 
 def build(recipe_paths_or_metadata, post=None, need_source_download=True,
-          build_only=False, notest=False, config=None, **kwargs):
+          build_only=False, notest=False, config=None, verify=True, **kwargs):
```

Comment:
I'm a little bit ambivalent on this, but for the sake of consistency, maybe this should be defined as noverify, to match notest.  Keeping all logic behaving the same like this makes things easier to keep in line.